keyboards/KeyHive/Maypad/Maypad.h

```diff
@@ -0,0 +1,20 @@
+#ifndef KB_H
```

Comment:
It appears you've generated these files with kbfirmware - their version of QMK is very old and basically not supported anymore. I would highly recommend you use the `./util/new_keyboard.sh` script to create the keyboard from the template already in this repo: https://docs.qmk.fm/#/hardware_avr?id=adding-your-avr-keyboard-to-qmk

Reply:
Awesome, thank you! Sorry, my first time here. I will generate it that way and make sure it all works! Thanks for the help

Reply:
No worries, it's just much easier on the reviewers (and you!) 😄